oci-library/oci-hol/java-management/set-up-of-management-agent/set-up-of-management-agent.md

```diff
@@ -284,12 +324,52 @@ Install Management Agent (If your host is Windows, skip to **For Windows** Secti
 
   ![image of modify agent settings page](/../images/fleet-modify-agent-settings-new.png)
 
+## Task 6: Verify application logging by Java Usage Tracker
+1. For the logging of applications to be visible, Java applications must be run again after the installation of the Management Agent. Run the following commands to run the HelloWorld application again:
+
+    ```
+    <copy>
+    javac HelloWorld.java
```

Comment:
IIRC, we have already compiled HelloWorld.java earlier. There is no need to compile again, we can omit javac "HelloWorld.java"

Reply:
Added the javac portion initially aiming to improve user understanding, as I thought it might be helpful to show that JMS can detect not only Java applications but also those from the compiler. Do you think it would be helpful in this case?

Reply:
Ok that is a valid point. In this case, probably can give a little further context:

_"Now that the Management Agent has been setup in your host, we will demonstrate the compiling of the HelloWorld.java again from the earlier lab, and then executing it and observe it from the JMS console"_

Something along this lines